README.md

```diff
@@ -33,7 +33,7 @@ Use the following table to verify you have the correct prerequisites to install
 		<td><a href="http://devdocs.magento.com/guides/v2.0/install-gde/prereq/apache.html">Apache</a></td>
 	</tr>
 	<tr>
-		<td>PHP ~5.5.22 or 5.6.x</td>
+		<td>PHP ~5.5.22,5.6.x or 7.x</td>
```

Comment:
Please specify supported PHP7 versions more precisely, according to http://devdocs.magento.com/guides/v2.0/install-gde/system-requirements.html